tools/profiler.md

```diff
@@ -24,24 +22,18 @@ Regarding latency, it is a whole topic in itself, but remember that we are talki
 
 ## Prerequisites
 
-Starting from Unity 2021.1, the Unity profiler can extend its features with Netcode for GameObjects [NetCode]. Netcode enables you to profile your game in the same tool you are already using for CPU, GPU, and memory.
-
-To enable profiling support for Netcode, install the `com.unity.multiplayer.tools` package from the [Unity Package Manager](../tools/install-tools.md). Then, you should see two additional modules in your profiler (**Window** > **Analysis** > **Profiler**):
-* Messages - Network messages sent to the transport
-* Objects - Activity related to, and grouped by, network objects
+Starting from Unity 2021.1, the Unity profiler can extend its features with Netcode for GameObjects [Netcode]. Netcode enables you to profile your game in the same tool you are already using for CPU, GPU, and memory.
```

Comment:
I think this section could be disambiguated a bit, as it seem that the word "Netcode" is being used to refer to NGO, multiplayer tools, and the network profiler in this paragraph. Maybe this could read:

> Starting from Unity 2021.1, the Unity profiler can extended with features to support profiling the network traffic of multiplayer games built using Netcode for Game Objects (NGO). These features are available as part of the Network Profiler which is included in the Multiplayer Tools package.  The Network Profiler allows you to profile the networking activity of your game in the same profiler that you use to profile CPU, GPU, and memory.

Reply:
Ah, yes, that is important and makes it clearer to me as well. 👍